lwa_antpos/reading.py

```diff
@@ -63,3 +69,16 @@ def read_antpos_yaml(filename=antposfile_yaml):
     df.set_index('antname', inplace=True)
 
     return df
+
+
+def update_antpos_etcd():
```

Comment:
um, are you sure you want everyone who imports this module to be able to change the etcd thing for everyone?

Reply:
Fair point. This is a convenience function, but setting etcd probably shouldn't be too convenient. I'll punt on this for now and note that we need to define controls on setting to etcd.